src/app/(proper_react)/redesign/(authenticated)/admin/feature-flags/page.tsx

```diff
@@ -37,16 +40,50 @@ export default async function FeatureFlagPage() {
             <th>Dependencies</th>
             <th>Allow List</th>
             <th>Wait List</th>
+            <th>Owner</th>
           </tr>
         </thead>
         <tbody>
           {data.map((item) => (
             <tr key={item.name}>
               <td>{item.name}</td>
-              <td>{String(item.is_enabled)}</td>
-              <td>{item.dependencies}</td>
-              <td>{item.allow_list?.join(", ")}</td>
-              <td>{item.wait_list?.join(", ")}</td>
+              <td>
+                <ToggleFlagEnabled
+                  id="isEnabled"
+                  name={item.name}
+                  isEnabled={item.is_enabled}
+                />
+                {item.is_enabled}
+              </td>
+              <td>
+                {" "}
```

Comment:
Q: Why does this need a leading space?

Reply:
This is something VSCode seems to autocomplete, it's really annoying. I'll remove.